index.html

```diff
@@ -1,24 +1,43 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Index</title>
+<title>Mount Sion CBS Secondary School</title>
     
     <link rel="stylesheet" href="style.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
-  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js"></link>
+  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js">
    <script src="//code.jquery.com/jquery-2.2.1.js"></script>
     
 </head>
  <body>
     
-<ul>
-  <li><a class="active" href="index.html">Home</a></li>
-  <li><a href="about.html">About</a></li>
-  <li><a href="history.html">History</a></li>
-  <li><a href="contact.html">Contact</a></li>
-</ul>
-    
-    
-    
+<img class="ui centered medium image" src="https://i.imgur.com/ajcswUA.jpg">
```

Comment:
I wouldn't be a fan of using `imgur` for the image, if you could download a copy of the image locally, create directory in the project `assets/images` and put the image in that dir. Till a full decision is made on where we will host images and media it is best we store them locally within the project.